solr/contrib/prometheus-exporter/conf/solr-exporter-config.xml

```diff
@@ -1053,6 +1053,51 @@
           </str>
         </arr>
       </lst>
+      <lst name="request">
+        <lst name="query">
+          <str name="path">/admin/zookeeper/status</str>
+        </lst>
+        <arr name="jsonQueries">
+          <str>
+            .zkStatus.ensembleSize as $value |
+            .zkStatus.mode as $mode |
+            {
+            name         : "solr_zookeeper_ensemble_size",
+            type         : "GAUGE",
+            help         : "See following URL: https://solr.apache.org/guide/cloud-screens.html#zk-status-view",
+            label_names  : [],
+            label_values : [],
+            value        : $value
+            }
+          </str>
+          <str>
+            .zkStatus.details[] as $object |
+            $object.host as $host |
+            $object.ok as $ok |
+            (if $object.clientPort != null and $ok then 1.0 else 0.0 end) as $value |
```

Comment:
We require both an "imok" response to the "ruok" 4lw, as well as the presense of clientPort, to work around the bug fixed in SOLR-15383 (zk node responds "imok" even if it is not connected to ensemble)